owl/EASE-OBJ.owl

```diff
@@ -26,6 +26,46 @@
     
 
 
+    <!-- http://www.ease-crc.org/ont/EASE-OBJ.owl#affects -->
+
+    <owl:ObjectProperty rdf:about="http://www.ease-crc.org/ont/EASE-OBJ.owl#affects">
+        <rdfs:subPropertyOf rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#hasConstituent"/>
+        <owl:inverseOf rdf:resource="http://www.ease-crc.org/ont/EASE-OBJ.owl#isAffectedBy"/>
+        <rdfs:domain rdf:resource="http://www.ease-crc.org/ont/EASE-OBJ.owl#Disposition"/>
+        <rdfs:range rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#Role"/>
+        <rdfs:comment>Associates a disposition to a role affected by it.</rdfs:comment>
+    </owl:ObjectProperty>
+    
+
+
+    <!-- http://www.ease-crc.org/ont/EASE-OBJ.owl#attracts -->
+
+    <owl:ObjectProperty rdf:about="http://www.ease-crc.org/ont/EASE-OBJ.owl#attracts">
```

Comment:
the attracts chain is useful to axiomatize objects that have some disposition.

Reply:
*attracts*: no, at least not intentionally. I think it is often ok to only have general properties and different contexts where the property is used. But well then the range/domain axioms must also be accordingly general which is not the case here. Hard to say if it should be more specific then Entity-Entity in case of a general attracts property.